doc/src/manual/interacting-with-julia.md

```diff
@@ -172,6 +172,7 @@ to do so).
 | `^Y`                | "Yank" insert the text from the kill buffer                                      |
 | `^T`                | Transpose the characters about the cursor                                        |
 | `^Q`                | Write a number in REPL and press `^Q` to open editor at corresponding stackframe or method |
+| `^/`, `^_`          | Undo |
```

Comment:
^/ isn't a valid control character (eg after subtracting 64 you get a negative number), however it appears that vt102 terminal emulators map it to emit ^_ for convience, see: https://apple.stackexchange.com/a/227286

From the terminals I tested on this is accurate that pressing ctrl-/ causes "^_" to appear in the console.